lite-locals.tf

```diff
@@ -1,5 +1,63 @@
+output "thor_max_jobs" {
```

Comment:
Is this output (and the next several lines) for debugging purposes or did you intend to have them remain for the final product?  If the latter, please update the documentation.

Reply:
These outputs are for debugging. Regarding you comments about the min_capacity and max_capacity in aks/locals.tf: all min_capacitys are 1. And, only the max_capacity for thorpool is variable because it is the only one that I can calculate.

Reply:
OK.  I will keep this conversation as unresolved until the debugging statements are removed.